src/main/java/com/revature/registry/model/Repository.java

```diff
@@ -17,10 +21,11 @@
     private int id;
 
     private String name;
+
     private String url;
 
     @ManyToOne
-    @JoinColumn(name = "project")
-    private Project project;
+    @JoinColumn(name = "organization_id")
+    private Organization organization;
```

Comment:
I believe this field might need a ManyToOne annotation.

Reply:
It's present on line 27 👍

Reply:
Oh, I don't know how I didn't see that. Perfect.